ministryofjustice/correspondence_tool_staff
# Ct 2427 filters layout

## Description
Update the filters layout and filters itself to some extent to match prototype and comments on the ticket
 
## Self-review checklist

 
### Screenshots
Yes but might change so will add later
![image](https://user-images.githubusercontent.com/22935203/65761251-57cb9280-e116-11e9-9ee9-06ac2ec827ec.png)
![image](https://user-images.githubusercontent.com/22935203/65761311-729e0700-e116-11e9-9be5-3aef03e821ed.png)
![image](https://user-images.githubusercontent.com/22935203/65761369-8ba6b800-e116-11e9-92ad-a33a90e80155.png)
![image](https://user-images.githubusercontent.com/22935203/65761385-98c3a700-e116-11e9-8ea6-4d8ec2ac0b89.png)
![image](https://user-images.githubusercontent.com/22935203/65761448-b264ee80-e116-11e9-9b91-c4434cf155d6.png)
![image](https://user-images.githubusercontent.com/22935203/65761510-d0caea00-e116-11e9-8303-ce331d340af8.png)
![image](https://user-images.githubusercontent.com/22935203/65761870-a0d01680-e117-11e9-8f22-680fa0782426.png)
 ![image](https://user-images.githubusercontent.com/22935203/65762097-19cf6e00-e118-11e9-9b58-27b79c9b4ca4.png)
![image](https://user-images.githubusercontent.com/22935203/65762141-323f8880-e118-11e9-9fea-4d95c6413166.png)



### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2427
 
### Deployment
Hopefully n/a
 
### Manual testing instructions
View case list (homepage) and check if filters in constantina look like prototype, and check notes on ticket/comments. For Search page the filters should match but also have the exemptions filter.
